dev/com.ibm.ws.security.kerberos.auth/resources/com/ibm/ws/security/kerberos/auth/internal/resources/KerberosMessages.nlsprops

```diff
@@ -0,0 +1,27 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.ws.security.kerberos.auth/resources/com/ibm/ws/security/kerberos/auth/internal/resources/KerberosMessages.nlsprops
+#COMPONENTPREFIX CWWKS
+#COMPONENTNAMEFOR WebSphere Application Server Security Kerberos Service
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+# -------------------------------------------------------------------------------------------------
+
+# For any new messages reserve a code from the nlsprops file in the com.ibm.ws.security.kerberos.java8 bundle
+
+KRB5_FILE_NOT_FOUND_CWWKS4345E=CWWKS4345E: The attribute [{0}] from the {1} element is configured to a file that does not exist at: {2}
```

Comment:
The reason `[{0}]` is double bracketed is because we use this convention in other places when referring to an attribute. So the message will resolve as: `The attribute [keytab] from the .....`

Variable `{2}` is a specific file location